tests/thread_local_tests/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13)
+
+project(thread_local_test)
```

Comment:
I suppose the final goal is to have some test target to be validated via AppVerif. May we rename/reorganize a little in order to have a possibility to extend target with more test cases (not only thread_local)?

Reply:
complexity_tests or verifier_tests or ?  
https://learn.microsoft.com/en-us/windows-hardware/drivers/devtest/application-verifier

Reply:
IMO `appverifier_tests` may be OK

Reply:
updated